README.md

```diff
@@ -1,32 +1,63 @@
 # sigstore-js
 
-⚠️ This project is not ready for general-purpose use! ⚠️
-
 A tool for signing and verifying signatures using JavaScript.
```

Comment:
Something to clarify this is specifically for sigstore
```suggestion
A JavaScript library and CLI for signing and verifying Sigstore signatures.
```

Reply:
👍